src/style/uomthesis.sty

```diff
@@ -205,18 +205,18 @@
 						\vspace{1cm}
 					ORCID: {\@orcid} \par
 						\vspace{3cm}
-					Submitted in total fulfilment of the requirements of the degree of \par
-					\bigskip
-					{\LARGE Doctor of Philosophy} \par
-				}
+                    Doctor of Philosophy \par
 					\vspace{2cm}
-				%\includegraphics[height=3cm]{\@univlogofile} \par

```

Comment:
No more logo?  Was this part of the requirements change?

Also, without the logo, the vertical spacing/layout looks a little strange.  I'd suggest either dropping the above `\vspace{2cm}` (with the `\LARGE` text change) or adjusting the whitespace so it looks more consistent across the entire page.